content/docs/command-reference/metrics/index.md

```diff
@@ -1,47 +1,104 @@
 # metrics
 
-A set of commands to add, manage, collect, and display project metrics:
-[add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove).
+A set of commands to manage, collect, and display _project metrics_:
+[show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff).
 
 ## Synopsis
 
 ```usage
 usage: dvc metrics [-h] [-q | -v]
-                   {show,add,modify,remove,diff} ...
+                   {show,diff} ...
 
 positional arguments:
   COMMAND
     show                Print metrics, with optional formatting.
-    add                 Mark a DVC-tracked file as a metric.
-    modify              Modify metric default formatting.
-    remove              Remove metric mark on a DVC-tracked file.
     diff                Show changes in metrics between commits
 ```
 
 ## Description
 
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
-metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
+metrics to track.
 
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+Metrics are project-specific floating-point or integer values e.g. AUC, ROC,
```

Comment:
I think it's better for types of metrics go before Description in this case (and in plots)

Reply:
at least the part that covers "we have two types ...."

Reply:
I tried doing this and it didn't make much sense but I'm making some changes so Types comes earlier in plots and also use the term "scalar metrics" in metrics so it's obvious there's a distinction. See ee87e99f (sorry, also has a bunch of small updates to plots refs — related to this PR)